scipy/signal/filter_design.py

```diff
@@ -34,18 +34,22 @@ class BadCoefficients(UserWarning):
 abs = absolute
 
 
-def findfreqs(num, den, N):
+def findfreqs(num, den, N, input='ba'):
```

Comment:
adding an `input=` parameter is another way to deal with https://github.com/scipy/scipy/issues/3259
but I would prefer making the function accept a system as the argument instead.
`findfreqs` isn't a great name to begin with, so can we think of a better name and call it like `important_filter_frequencies(system, N)` instead?